### DISCUSSION TITLE
有关4.1进度要求的疑问

### DISCUSSION THREAD

**Discussion by ghost** [liusy58/NJU_AdvancedProgramming_SP22#201] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/100082486/168476255-89d682c3-49ee-4596-9343-c1eec01495cd.png)
原文写`按照自己的进度选择 1 和 2 中一个或者多个来实现 `。如果我实现了`单位发起、受到攻击时, 需要绘制出相应的效果：被攻击显示和血量显示`和`2. 对于远程单位, 还需要表示出它在攻击哪个或者哪些敌方单位：绘制一道光束之类`，是否可以认为我`实现了1和2中的多个`，能拿到完全的分数？
`

**Comment by liusy58** [comment, marked as answer]
可以Q戳我
